monitoring/timedrift.go

```diff
@@ -203,7 +243,7 @@ func (c *timeDriftChecker) getTimeDrift(ctx context.Context, node serf.Member) (
 	// Finally calculate the time drift between this and the specified node
 	// using formula: T2 - T1Start - Latency.
```

Comment:
The comment needs updating too then.

Reply:
It's actually a good catch, I made a small semantic difference in the formula. I've now switched it to `now - t2 - latency` such that now - t2 should be a positive duration, and the -latency should adjust back to estimate time due to latency.